Source/Scene/Cesium3DTile.js

```diff
@@ -389,8 +378,11 @@ define([
      * DOC_TBA
      */
     Cesium3DTile.prototype.update = function(owner, frameState) {
-        applyDebugSettings(this, owner, frameState);
-        this._content.update(owner, frameState);
+        if (this.selected) {
```

Comment:
In practice, it will not matter, but if you want to be cool and optimize to avoid a cache miss change this to:
```javascript
        if (this.selected) {
            this.selected = false;
            applyDebugSettings(this, owner, frameState);
            this._content.update(owner, frameState);
        }
```
Otherwise, we risk `this._content.update` kicking `selected` out of a register (almost most certainty) and maybe even out of L2 (if, for example, it called into WebGL).